spec/lib/puppet_litmus/rake_tasks_spec.rb

```diff
@@ -2,6 +2,8 @@
 
 require 'spec_helper'
 require 'rake'
+load File.expand_path('../../../lib/puppet_litmus/inventory_manipulation.rb', __dir__)
+include PuppetLitmus::InventoryManipulation # rubocop:disable Style/MixinUsage
```

Comment:
These two lines are in here because they are undeclared dependencies for this test file. The full suite runs because previous test file shave loaded these, but if you try to run this file on it's own these are missing.